### PR TITLE
Airport Challenge Attempt

### PR DESCRIPTION
**Pull request by Hannah-Frost**

### CODE REVIEW

File: lib/airport.rb [hunk 1]
```diff
@@ -8,4 +8,10 @@ def initialize
   def land_plane(plane)
     @planes << plane
   end
+
+  def plane_takeoff
+    @planes[0]
```
Comment (tsankhalpara) [inline lib/airport.rb]: would using planes.pop be more useful to use to help refactor the code

File: lib/airport.rb [hunk 2]
```diff
@@ -14,4 +14,8 @@ def plane_takeoff
     @planes.delete_at(0)
   end
 
+  def empty?
```
Comment (tsankhalpara) [inline lib/airport.rb]: what was the reason for adding a definition for empty instead of full?

File: lib/airport.rb [hunk 3]
```diff
@@ -1,26 +1,52 @@
...
   end
 
+  private
+
   def empty?
     @planes.empty?
   end
 
+  def full?
+    @planes.count.positive?
```
Comment (tsankhalpara) [inline lib/airport.rb]: What was the reason for using this instead of comparing it to capacity?

File: lib/airport.rb (hunk 1 above)
Comment (Hannah-Frost) [reply lib/airport.rb]: It didn't occur to me to use `.pop` but I did end up using `.delete(plane)` instead. In the future I will use `.pop` when applicable though. :)

File: lib/airport.rb (hunk 2 above)
Comment (Hannah-Frost) [reply lib/airport.rb]: Good point! It was initially for clarification but it makes more sense to only have one (also makes it neater). I will change that now.

File: lib/airport.rb (hunk 3 above)
Comment (Hannah-Frost) [reply lib/airport.rb]: In my new code this is fixed:
```
  def full?
    @planes.count >= @capacity
  end
```
Thank you for your help!

**Pull request closed without merging**